### DISCUSSION TITLE
MacOS Menu Bar: "Zen" instead of "Zen Browser"

### DISCUSSION THREAD

**Discussion by w-lkchen** [zen-browser/desktop#2238] [Ideas]
I really don't like that in the menu bar on MacOS the app name shows up as "Zen Browser" but everything else shows "Zen" (for example: " About Zen")
It would look a lot cleaner to have it just be "Zen" in the menu bar.
In other browsers its the same, it doesn't say "Arc Browser", or "Google Chrome Browser"
I think this should just be a simple edit to the Info.plist

Thanks for reading :)

**Comment by flyespresso** [comment]
+1 It'd get valuable menu bar space back!

**Reply by w-lkchen** [reply]
Yes! and it would also sound and look better :)

**Comment by Richard-Woessner** [comment]
I like this

**Comment by w-lkchen** [comment]
Looks like it was changed,
thank you :)
